x-pack/plugins/index_management/public/application/app_context.tsx

```diff
@@ -56,6 +57,7 @@ export interface AppDependencies {
   docLinks: DocLinksStart;
   kibanaVersion: SemVer;
   theme$: Observable<CoreTheme>;
+  toasts: NotificationsSetup['toasts'];
```

Comment:
We already have `services.notificationService` in the app context, I think we could just re-use it, it's like a wrapper around toasts.